modules/custom/openy_search/SMOKE_TESTS.md

```diff
@@ -0,0 +1,81 @@
+# Welcome to Grid Content smoke tests documentation
```

Comment:
`Grid Content` is not related to this module, looks like copy/paste from different documentation

Reply:
addressed https://github.com/ymcatwincities/openy/pull/2535/commits/f53eaddc5655b0974cf125f8d32468d4bb7788a4